conf/ocp_version/ocp-4.4-ga-config.yaml

```diff
@@ -7,7 +7,10 @@ RUN:
 DEPLOYMENT:
   ocp_url_template: "https://mirror.openshift.com/pub/openshift-v4/clients/ocp/{version}/{file_name}-{os_type}-{version}.tar.gz"
   installer_version: "4.4-ga"
-  ocp_channel: "stable"
+  ocp_channel: "fast"
```

Comment:
how will you come to know that 4.4 moved to Stable? Do you get notification?

Reply:
We will need to check manually.  As was mentioned in mail thread one of web app can easily show you now here:
https://openshift-channels.robszumski.com/ 

Or we can use the command I mentioned in mail thread.